docassemble/AppealsCivilDocketingStatement/data/questions/civil_docketing_statement.yml

```diff
@@ -45,11 +45,16 @@ code: |
   interview_short_title = interview_metadata['civil_docketing_statement']['title']
 ---
 mandatory: True
-id: interview_order_civil_docketing_statement
+id: main_order_civil_docketing_statement
 code: |
-  # This is a placeholder to control logic flow in this interview
-  appeals_basic_questions_intro_screen 
```

Comment:
I think the `main_order` should still have `appeals_basic_questions_intro_screen`, just not the `interview_order`. 

Once that's in again, this'll be good to go!